feature/achievements/src/test/java/io/github/droidkaigi/confsched2023/achievements/AchievementsScreenTest.kt

```diff
@@ -30,6 +38,11 @@ class AchievementsScreenTest {
     @Inject
     lateinit var achievementsScreenRobot: AchievementsScreenRobot
 
+    @Before
+    fun setup() {
+        LottieTask.EXECUTOR = Executor(Runnable::run)
```

Comment:
Maybe you should restore the executor after test to isolate the tests 👀

Reply:
@takahirom 
I've checked if this setup affects other test classes.
To invesigate that, I created AchievementsScreenTest2 copying original one except setUp function. As a result, all the test cases in AchievementsScreenTest2 didn't capture achivement highlight correctly because LottieTask.EXECUTOR was not set up for tests.
So, I assume changing LottieTask.EXECUTOR is confined in its test class and tearDown isn't really necessary.
However, if you are concerned about other test cases in the same test class, which doesn't need to set up LottieTask.EXECUTOR, I'd make setup in each test cases like below. But I feel it's kind of hacky.
```
fun checkHighlightImageA() {
    achievementsScreenRobot {
        LottieTask.EXECUTOR = Executor(Runnable::run)
        setupSavedAchievement(ArcticFox)
        setupScreenContent()
        clickAchievementImageA()
        checkScreenCapture()
        LottieTask.EXECUTOR = Executors.newCachedThreadPool()
    }
}
```

Reply:
As far as I know static variable is used across the tests. We have some options to do it. 
How about adding it here?
https://github.com/DroidKaigi/conference-app-2023/blob/b53057115722cacf1a7fcd6eee2786d1a0267516/core/testing/src/main/java/io/github/droidkaigi/confsched2023/testing/RobotTestRule.kt#L64

Reply:
@takahirom 
I wasn't aware of that system.
I added LottieTestRule.